doc/source/tracking/tracking.rst

```diff
@@ -0,0 +1,48 @@
+Tracking icepyx Usage
+======================
+
+How is icepyx being used by the ICESat-2 data user community?
+
+
+Projects and Organizations
+--------------------------
+Projects and organizations that use icepyx (this list is still being populated as of Nov 2020)
```

Comment:
This is great! Would you like to also include a statement encouraging folks to add their organization / a link to their project if they are using icepyx?